erpnext/manufacturing/doctype/production_plan/production_plan.py

```diff
@@ -47,7 +47,7 @@ def add_so_in_table(self, open_so):
 				'sales_order': data.name,
 				'sales_order_date': data.transaction_date,
 				'customer': data.customer,
-				'grand_total': data.grand_total
+				'grand_total': data.base_grand_total
```

Comment:
why are we mapping Grand Total against Base Grand Total? They are 2 different things.